manifests/base.pp

```diff
@@ -74,15 +56,14 @@
       # all configuration is under the rsyslog.d directory
       |EOT
 
-    $_require = $rsyslog::manage_package ? {
-      false   => undef,
-      default => Package[$rsyslog::package_name],
-    }
     file { $rsyslog::config_file:
-      ensure  => 'file',
+      ensure  => file,
```

Comment:
Apparently the style guide says to do this.  I don't really care, but at least it's consistent with how `ensure => directory` doesn't single quote `directory` in the rest of this manifest.  (That and vim puppet highlights the bare words `file` and `directory`!)